aether-kernel/aether/kernel/api/entity_extractor.py

```diff
@@ -436,7 +481,9 @@ def extract_entity(entity_type, entities, requirements, data, entity_stub):
     # make empty stubs for our expected outputs. One for each member
     # identified in count process
     entities[entity_type] = [
-        {field: None for field in entity_stub.keys()} for i in range(count)]
+        {field: None for field in entity_stub.keys()
+            if field in requirements.get(entity_type)}  # make sure there are mappings
```

Comment:
I know that this is me but don't you think that it's more clear to read this way?
```python
{
    field: None
    for field in entity_stub.keys()
    if field in requirements.get(entity_type)  # make sure there are mappings
}
```

Reply:
:+1: for readability